orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBSourceSettings.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb
+
+//#source-settings
+final case class OrientDBSourceSettings(maxPartitionSize: Int = Runtime.getRuntime.availableProcessors(),
+                                        maxPoolSize: Int = -1,
+                                        skip: Int = 0,
+                                        limit: Int = 10,
+                                        dbUrl: String,
```

Comment:
Declaring non-optional parameters after optional parameters doesn't sound good to me.